subworkflows/local/germline_variant_calling.nf

```diff
@@ -69,8 +69,8 @@ workflow GERMLINE_VARIANT_CALLING {
             sample = meta.sample
             new_intervals = intervals.baseName != "no_intervals" ? intervals : []
             id = new_intervals ? sample + "_" + new_intervals.baseName : sample
-            new_new_meta = [ id: id, sample: meta.sample, gender: meta.gender, status: meta.status, patient: meta.patient ]
-            [new_new_meta, cram, crai, new_intervals]
+            new_meta = [ id: id, sample: meta.sample, gender: meta.gender, status: meta.status, patient: meta.patient ]
+            [new_meta, cram, crai, new_intervals]
```

Comment:
Do you think this would work, is it easier to read, or not even?
```suggestion
            [[ id: id, sample: meta.sample, gender: meta.gender, status: meta.status, patient: meta.patient ]
 cram, crai, new_intervals]
```

Reply:
let me try

Reply:
seems to work (tried in another subworkflow now)

Reply:
maybe not easier to read but less confusing I think, no?